lib/services/database_service.dart

```diff
@@ -190,10 +235,13 @@ class DatabaseService {
 
       return UpdateNewConversationPage(suggestions: leaguers);
     } catch (error, trace) {
-      return AddProblem.from(
-          message: error.toString(),
-          traceString: trace.toString(),
-          type: ProblemType.retrieveLeaguers);
+      return AddProblem(
+        problem: RetrieveLeaguersProblem.by(
```

Comment:
Should this be "UpdateNewConversationPageProblem"?

Reply:
Another good find! In the latest commit